sdk/core/core/src/az_core.c

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_assert.h>
```

Comment:
what's this header for? is it to test everything in one TU? Is it a PCH file?

Reply:
Currently, it's just one file that test compilation.